### ISSUE TITLE
 Question about the result.

### ISSUE THREAD

**Issue by cowry5** [RuihongQiu/SRGNN_PyG#4]
I found that the diginetica's result of the original paper is 27. But this reposit is 17. Do you know what is going on?

**Comment by RuihongQiu** [comment]
According to the Table 2 in the paper, the result of Diginetica is 17.59. If this number is incorrect, would you please provide a link or a screenshot?

**Comment by cowry5** [comment]
This is the picture I took in the original paper.  https://arxiv.org/pdf/1811.00855v3.pdf

![image](https://user-images.githubusercontent.com/32674827/68538793-e50c3300-03b4-11ea-9afd-f0d979d81f4e.png)

But I just found out that the author has released the v4, the latest version, and I saw the v3 before.
In the paper v4, you are correct, the MRR@20 result of Diginetica is 17.59.

So I don't know why the author modified his results...
